presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/ReorderJoins.java

```diff
@@ -602,88 +599,16 @@ private JoinEnumerationResult createJoinEnumerationResult(PlanNode planNode)
      * This class represents a set of inner joins that can be executed in any order.
      */
     @VisibleForTesting
-    static class MultiJoinNode
+    static class ReorderJoinsMultiJoinNode
```

Comment:
Since the only difference is the `toMultiJoinNode`  method, why don't we just make this method a member of ReorderJoins and just make it return a regular `MultiJoinNode`

Reply:
changed. pls check the [commit](https://github.com/Haritha-Koloth/presto/pull/1/commits/8de7fc4b3bcb837e403728b64154c04a4d2abefb). Is this what u meant ?

Reply:
No I meant, we don't need to create the subclass ReorderJoinsMultiJoinNode or GroupInnerJoinsMultiJoinNode